### ISSUE TITLE
Proposal: Align Releases to SBL-Versions (as long as both extensions coexist)

### ISSUE THREAD

**Issue by twiro** [symphonists/association_field#19]
Thank you very much  for the 1.0.0-Release @nilshoerrmann !
About time to get this out of beta ;)

I would like to propose to align version numbers (and therefore (basic) commits/fixes) of this extension to the ones of the original [SBL-Repo](https://github.com/symphonycms/selectbox_link_field) as long as both versions coexist _(which ain't a perfect state, as the whole idea of association field was to enhance **and replace** SBL, but seems to be the way things are for now (Symphony 2.X)_.

I have checked the additions to the 1.0.0-Release and have 4 additional SBL-Commit-Backports left, that seem to not be included yet:

- https://github.com/twiro/association_field/commit/f344347c05e388a2a1b0254a07fd6458c8b6bbc8
- https://github.com/twiro/association_field/commit/7db7d331dbf5031f7a8aa7f53a4da0c53831768c
- https://github.com/twiro/association_field/commit/72f22379ac6fdad16df968a55124fb19acf322fc
- https://github.com/twiro/association_field/commit/308db3895c24a4a4b7d7ae271f781e9b74f52e12

What about merging these, adding a readme _(at least a short one - could help with that one if no native english speaker wants to jump in)_ and release a 1.3.3-Version that matches the current latest SBL-Version?

Any thoughts @brendo , @nilshoerrmann ?

Maybe @jonmifsud 's filtering additions could also be included in that release:

- https://github.com/symphonists/association_field/pull/18

**Comment by twiro** [comment]
> _Maybe @jonmifsud 's filtering additions could also be included in that release_

Just noticed that the whole "association filters"-concept probably won't be implemented before Symphony 3.0.0. ( https://github.com/symphonycms/symphony-2/issues/2322#issuecomment-74543417 )

So that PR wouldn't need to be part of a possible 1.3.3-Release.
Am I getting this right @jonmifsud ?

_(Though I already tested your filtering-branch in a current S.2.6.2-project and it solved some javascript-problems that the regular 1.0.0.beta had ( https://github.com/hananils/association_ui_selector/issues/47 ) Will need to check if these bugs are fixed in the new 1.0.0-release )_

**Comment by jonmifsud** [comment]
@twiro yes correct - but I might be re-evaluating the core requirement / implementation and maybe find a workaround. I've submitted a separate PR for filtering via delegate which might mean I can move the current 'filtering branch' into a separate extension which handles that exclusively.
